kofta/src/vscode-webview/components/VoiceSettings.tsx

```diff
@@ -7,9 +7,11 @@ import { Button } from "./Button";
 import { MuteKeybind, PTTKeybind } from "./keyboard-shortcuts";
 import { VolumeSlider } from "./VolumeSlider";
 
-interface VoiceSettingsProps {}
+interface VoiceSettingsProps {
+  className?: string;
+}
 
-export const VoiceSettings: React.FC<VoiceSettingsProps> = ({}) => {
+export const VoiceSettings: React.FC<VoiceSettingsProps> = ({ className }) => {
```

Comment:
instead of passing a className to VoiceSettings, just wrap the component in a `<div />` with the `className` you want to apply